docs/iris/src/index.rst

```diff
@@ -44,11 +38,61 @@ Interoperability with packages from the wider scientific Python ecosystem comes
 from Iris' use of standard NumPy/dask arrays as its underlying data storage.
 
 Iris is part of SciTools, for more information see https://scitools.org.uk/.
+For **Iris 2.4** and earlier documentation please see the
+:link-badge:`https://scitools.org.uk/iris/docs/v2.4.0/,"legacy documentation",cls=badge-info text-white`.
+
+
+.. panels::
+    :container: container-lg pb-3
+    :column: col-lg-4 col-md-4 col-sm-6 col-xs-12 p-2
+
+    Install Iris to use ir for development.
+    +++
+    .. link-button:: installing_iris
+        :type: ref
+        :text: Installing Iris
+        :classes: btn-outline-primary btn-block
+    ---
+    View the gallery that includes python code used to create it.
+    +++
+    .. link-button:: sphx_glr_generated_gallery
+        :type: ref
+        :text: Gallery
+        :classes: btn-outline-primary btn-block
+    ---
+    Find out what has recently been added to Iris, or soon will be.
```

Comment:
Not sure about the "soon will be", as this won't make sense for the released docs.  Also this is currently pointing to a list of all the whatsnews.  Should it just be the whatsnew for the current version?  If intended to point to all versions, maybe "Find out what has changed in Iris" would be more all-encompassing.

Reply:
To be fair, all of it has been recently added to Iris, but only some of it has been released so far.

Reply:
In fact, some things have be _removed_ from Iris....

Reply:
how about:

> "Find out what has recently changed in Iris."

Reply:
>"Find out what has recently changed in Iris."

I'm in favour of this